doc/ref/states/ordering.rst

```diff
@@ -24,23 +24,18 @@ dependencies via the `requisite` system.
 State Auto Ordering
 ===================
 
-.. versionadded: 0.17.0
```

Comment:
Why are you removing this?

Reply:
According to (modified) version history, I would never ever have done such a silly thing! ;)